examples/large_models/deepspeed/bloom/custom_handler.py

```diff
@@ -0,0 +1,119 @@
+import logging
+from abc import ABC
+
+import torch
+import transformers
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from ts.context import Context
+from ts.handler_utils.distributed.deepspeed import get_ds_engine
+from ts.torch_handler.distributed.base_deepspeed_handler import BaseDeepSpeedHandler
+
+logger = logging.getLogger(__name__)
+logger.info("Transformers version %s", transformers.__version__)
+
+
+class TransformersSeqClassifierHandler(BaseDeepSpeedHandler, ABC):
+    """
+    Transformers handler class for sequence, token classification and question answering.
+    """
+
+    def __init__(self):
+        super(TransformersSeqClassifierHandler, self).__init__()
+        self.max_length = None
+        self.tokenizer = None
+        self.initialized = False
+
+    def initialize(self, ctx: Context):
+        super().initialize(ctx)
+        self.max_length = int(ctx.model_yaml_config["handler"]["max_length"])
+        self.model = AutoModelForCausalLM.from_pretrained(
+            "bigscience/bloom-7b1", use_cache=False
```

Comment:
@lxning could read the model name from config

Reply:
the main diff b/w bloom example and opt example s to demo the model can be either downloaded via HF model name or via dowmload tool.